setup.py

```diff
@@ -47,6 +52,8 @@
         "proto-plus >= 1.10.1",
         "google-cloud-storage >= 1.26.0, < 2.0.0dev",
         "google-cloud-bigquery >= 1.15.0, < 3.0.0dev",
+        "tensorflow-cpu",
+        "grpcio-testing",
```

Comment:
This is a dependency for test. I tried putting it in testing/constraints-3.x.txt but didn't seem to work. Where is the best place for it?

Reply:
I went ahead an opened this PR to handle these dependencies: https://github.com/googleapis/python-aiplatform/pull/369

Once it is merged, update your branch.

Reply:
#369